rust/libdnf-sys/build.rs

```diff
@@ -2,5 +2,36 @@ use anyhow::Result;
 
 fn main() -> Result<()> {
     system_deps::Config::new().probe()?;
+    use cmake::Config;
+    let libdnf = Config::new("../../libdnf")
+        // Needed for hardened builds
+        .cxxflag("-fPIC")
+        // I picked /usr/libexec/rpm-ostree just because we need an
+        // arbitrary path - we don't actually install there.
```

Comment:
It'd default to `/usr/local` then I think; the idea here is to use a should-be-nonexistent path so that if it ever shows up at runtime we know it's a bug.  (And actually one thing we've broken is the libdnf translations, but ostree and rpm-ostree aren't translated, and we don't even expose a way to change the output language for our DBus API so...)